scheduler/services/resource/ocs_resource_service.py

```diff
@@ -146,14 +150,15 @@ def __init__(self, sites: FrozenSet[Site] = ALL_SITES):
             # This will put both the IFU and the FPU barcodes available on a given date as Resources.
             # Note that for the IFU, we need to convert to a barcode, which is a Resource.
             # This is a bit problematic since we expect a list of strings of Resource IDs, so we have to take its ID.
-            self._load_csv(site, f'GMOS{suffix}_FPUr201789.txt',
-                           lambda r: {self._itcd_fpu_to_barcode[site][r[0].strip()].id} | {i.strip() for i in r[1:]})
+            self._load_csv(site,
+                           lambda r: {self._itcd_fpu_to_barcode[site][r[0].strip()].id} | {i.strip() for i in r[1:]},
+                           name=f'GMOS{suffix}_FPUr201789.txt')
```

Comment:
Isn't this code now mostly repeated in `sources.py`? I'm confused.

Reply:
Yes, I'll do something to reutilize this code